fetch-ledger-develop-image/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM ubuntu:18.04
 
+RUN apt-get update
```

Comment:
docker build fails for me without this, when trying to apt-get install vim and git